highlevel/robot/controller/motion/position.py

```diff
@@ -5,11 +5,29 @@
 
 from highlevel.logger import LOGGER
 from highlevel.robot.entity.configuration import Configuration
-from highlevel.robot.entity.type import Millimeter
-from highlevel.util.odometry import OdometryFunc
+from highlevel.robot.entity.type import Millimeter, Radian, Tick, MillimeterPerSec
+from highlevel.util.filter.odometry import OdometryFunc
 from highlevel.util.probe import Probe
 
 
+def tick_to_mm(tick: Tick, ticks_per_revolution: Tick,
```

Comment:
these 2 functions look like util functions that could be extracted in the util package